tests/system/conftest.py

```diff
@@ -145,6 +145,16 @@ def session() -> Generator[bigframes.Session, None, None]:
     session.close()  # close generated session at cleanup time
 
 
+@pytest.fixture(scope="session")
+def session_us_east5() -> Generator[bigframes.Session, None, None]:
```

Comment:
Did we use it anywhere?

Reply:
forgot to remove in this PR, but will use in PR of adding tests.